fast-accountlookup-service/src/utils/dateUtils.ts

```
@@ -0,0 +1,62 @@
+  static isDateInFuture(date: string): boolean {
+    const inputDate = moment.tz(date, 'DD/MM/YYYY', this.timezone);
+    const currentDate = moment.tz(this.timezone);
+    return inputDate.isAfter(currentDate);
+  }
+
+  static isDateInPast(date: string): boolean {
+    const inputDate = moment.tz(date, 'DD/MM/YYYY', this.timezone);
+    const currentDate = moment.tz(this.timezone);
+    return inputDate.isBefore(currentDate);
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)